tests/test_core.py

```diff
@@ -642,8 +686,11 @@ def test_interpolate_extra_cols():
     df2 = df.copy()
     df2.interpolate(2007)
 
-    # assert that interpolation didn't change any data
-    assert_iamframe_equal(df, df2)
+    # interpolate should work as if this is a new index
```

Comment:
Not clear to me what this comment means...

Reply:
have cleared it up